outputter/json_test.go

```diff
@@ -0,0 +1,115 @@
+package outputter
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/aquasecurity/bench-common/check"
+)
+
+type mockFile struct {
+	fail bool
+}
+
+func (mf *mockFile) Handle(data string) error {
+	if mf.fail {
+		return fmt.Errorf("Failed to Handle data")
+	}
+	return nil
+}
+
+type mockConverter struct {
+	fail bool
+}
+
+var errorTestFailedToConvert = fmt.Errorf("Failed to ConvertToJSON data")
+
+func (cj *mockConverter) ConvertToJSON(controls *check.Controls) ([]byte, error) {
```

Comment:
IMO, I think this mock is too much of an abstraction from the real ConvertToJSON method. Instead what I would do is to pass varying inputs for the `controls` parameter to the real ConvertToJSON method and see it react differently under happy and sad paths.

Reply:
Also on this another comment: https://github.com/aquasecurity/bench-common/pull/52/files#r340948473